website/templates/project/modal_add_contributor.mako

```diff
@@ -71,7 +71,7 @@
                                             <!-- height and width are explicitly specified for faster rendering -->
                                             <img data-bind="attr: {src: contributor.gravatar_url}" height=35 width=35 />
                                         </td>
-                                        <td width="75%">
+                                        <td width="75%" style="word-break: break-all">
```

Comment:
No inline style please

Reply:
Should these be `break-all` or `normal`? Probably should check with @laurenrevere who just went through a journey on breaking things at words. Also, yes, use css for the style on both of these as Alex suggested.